chain-impl-mockchain/src/evm/mod.rs

```diff
@@ -83,7 +111,7 @@ fn serialize_address(
     bb: ByteBuilder<EvmTransaction>,
     caller: &Address,
 ) -> ByteBuilder<EvmTransaction> {
-    bb.u8(0).bytes(caller.as_fixed_bytes())
```

Comment:
do you know why was the 0 there? Is it some sort of tag?

Reply:
Not sure, I am looking how it is implemented in the `Aurora` project.

Reply:
I have not found any similar serde and even any similar `EvmTransaction` struct that we have. So it is just our custom struct with our custom serde format. Therefore, IMO this one byte seems to be redundant for the `Address` serialization.

Reply:
I'm pretty sure that it was an error when refactoring the code into the `serialize_address` function. The `Address` de/serialization is handled as fixed 20 bytes (it is an alias for the `H160` primitive type), so there's no need to use tags.